src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_actions.py

```diff
@@ -167,71 +162,3 @@ def _create_image_instance(publisher, offer, sku, version):
         'sku': sku,
         'version': version
     }
-
-
-def _handle_container_ssh_file(**kwargs):
```

Comment:
I do agree with moving this stuff out of _actions. Ideally this file will go away!